agent/main/lib/Browser.ts

```diff
@@ -312,9 +313,11 @@ export default class Browser extends TypedEventEmitter<IBrowserEvents> implement
   }
 
   public setUserDataDir(): void {
+    // TODO weird behaviour if userDataDir is not set but --user-data-dir is passed
```

Comment:
There are times you want to point at a specific location (eg, you've saved off a base profile). Assuming that's still true, I think we should still allow you to do this, and for now just ignore if they break it. At some point it would be nice to copy the dir or tell them somehow.